### ISSUE TITLE
wish: keep a changelog

### ISSUE THREAD

**Issue by markstos** [containers/podman-compose#499]
It would help people evaluating upgrading if a changelog was kept. For example, one of the last releases had a major version upgrade. This implies possibly breaking changes, but documentation is found of what the breaking changes are.

The easiest way to keep a Changelog is to use the Github "Releases" feature.

**Comment by muayyad-alsadi** [comment]
We have two major releases 

0.1.x which predates inter-container communication in legacy podman versions < 3.2.0

1.0.x which uses new `podman network create` and dns plugin

And the current devel version 1.0.4 which will become 1.1.x which puts all containers in a single pod

**Comment by rugk** [comment]
I agree with the OP that at least some simple changelog, if not e better one, would likely reduce a lot of support questions and effort for you as a maintainer. The thing is if stuff is not fully documented, _especially_ breaking changes, users will have to ask what happened.
With documented, I mean a changelist, upgrade guides, etc.

So a small Changelog could mean more effort once for writing it, but less support later for answering all the tickets etc.

**Comment by rugk** [comment]
BTW a technical addendum: If you wanna generate a [Changelog from commits e.g.](https://github.com/conventional-changelog/standard-version) (though now deprecated as I just see, but there are alternatives).

**Comment by p12tic** [comment]
The changelogs are now kept explicitly and are in docs/ subdirectory.
